andela/wger_alphas
# Ft add fitbit support 142465511

#### What does this PR do?
This pull request enables a user to sync the wger workout manager with their fitbit account and as a result data can be transferred via the Fitbit API
#### Description of Task to be completed?
A user should be able to pull in weight and exercise information from Fitbit.
#### How should this be manually tested?
This task can be manually tested by going to the weight overview page and selecting the fitbit sync button. This action will redirect you a page which has information regarding the next steps a user should take. After pressing the connect button a user is provided with login and consent pages from the Fitbit website and after this the user's data is automatically synced to our databases.
#### What are the relevant pivotal tracker stories?
#142465511
